### PR TITLE
I just pulled the Draco out the Rari

### PR DESCRIPTION
**Pull request by Holli-Git**
#### Summary
Content "Draco pistol additions"

#### Purpose of change

I noticed that the mini draco we had in game seemed to be a mismash of all the dracos. I originally wanted to just fix this, but realized all the draco variants can fit in the game, so I did that.

#### Describe the solution

Adds in 3 new guns, adjusts 1. 
Draco: Following the ID name, I changed it to the biggest draco.
Mini draco: A smaller version of the draco
Micro draco: An even smaller version
NAK9: Essentially a draco in 9mm

#### Describe alternatives you've considered

Just adjusting the draco

#### Testing
Spawning:
image
image
image

#### Additional context

image

### CODE REVIEW

File: data/json/items/gun/9mm.json [hunk 1]
```diff
@@ -1313,5 +1313,58 @@
...
     "melee_damage": { "bash": 9 }
+  },
+  {
+    "id": "draco_9mm",
+    "//": "Based off of https://www.centuryarms.com/draco-nak9.html",
+    "copy-from": "pistol_base",
+    "looks_like": "draco",
+    "type": "GUN",
+    "name": { "str": "Glock AK pistol" },
+    "description": "A 9mm AK pistol that accepts glock magazines.  Has an 11.14\" barrel.",
```
Comment (TheShadowFerret) [inline data/json/items/gun/9mm.json:1324]:
```suggestion
    "description": "A 9mm AK pistol that accepts Glock magazines.  Has an 11.14\" barrel.",
```

File: data/json/items/gun/9mm.json [hunk 2]
```diff
@@ -1313,5 +1313,58 @@
...
+    "variant_type": "gun",
+    "variants": [
+      {
+        "id": "draco_9mm",
+        "name": { "str": "NAK 9" },
+        "description": "The NAK 9 is a Century Arms imported gun.  It is essentially a shortened AK with no stock, chambered in 9mm and accepting glock magazines, with an 11.14\" barrel."
```
Comment (TheShadowFerret) [inline data/json/items/gun/9mm.json:1330]:
```suggestion
        "description": "The NAK 9 is a Century Arms imported gun.  It is essentially a shortened AK with no stock, chambered in 9mm and accepting Glock magazines, with an 11.14\" barrel."
```

File: data/json/items/gun/762.json [hunk 3]
```diff
@@ -213,7 +215,97 @@
...
+    "looks_like": "draco",
+    "type": "GUN",
+    "name": { "str": "Tiny AK pistol" },
+    "description": "Essentially a semi-auto AK with a short barrel and no stock, this unwieldy \"pistol\" has seen some popularity with civilian shooters and gangbangers alike.  This one has a 6.25\" barrel",
+    "variant_type": "gun",
+    "variants": [
+      {
+        "id": "draco_mini",
+        "name": { "str": "Micro Draco" },
+        "description": "The Draco series of guns are guns imported by Century Arms.  They're essentially semi-auto AKs, but with shorter barrels and no stock.  While advertised as compact guns, they're more often than not range toys.  This one is a Micro Draco, the smallest, with a 6.25\" barrel.  It has distinctive, oddly curvy handguard."
```
Comment (TheShadowFerret) [inline data/json/items/gun/762.json:278]:
```suggestion
        "description": "The Draco series of guns are guns imported by Century Arms.  They're essentially semi-auto AKs, but with shorter barrels and no stock.  While advertised as compact guns, they're more often than not range toys.  This one is a Micro Draco, the smallest, with a 6.25\" barrel.  It has a distinctive, oddly curvy handguard."
```

File: data/json/items/gun/762.json [hunk 4]
```diff
@@ -174,26 +174,28 @@
...
+    "name": { "str": "AK pistol" },
+    "description": "Essentially a semi-auto AK with a short barrel and no stock, this unwieldy \"pistol\" has seen some popularity with civilian shooters and gangbangers alike.  This one has a 12.5\" barrel.",
     "variant_type": "gun",
     "variants": [
       {
         "id": "draco",
-        "name": { "str": "Mini Draco \"pistol\"", "str_pl": "Mini Draco \"pistols\"" },
-        "description": "Essentially a semi-auto AK with a short barrel and no stock, this unwieldy \"pistol\" has seen some popularity with civilian shooters and gangbangers alike."
+        "name": { "str": "Draco" },
+        "description": "The Draco series of guns are guns imported by Century Arms.  They're essentially semi-auto AKs, but with shorter barrels and no stock.  While advertised as compact guns, they're more often than not range toys.  This one is a Draco, the largest, with a 12.5\" barrel"
```
Comment (TheShadowFerret) [inline data/json/items/gun/762.json:188]:
```suggestion
        "description": "The Draco series of guns are guns imported by Century Arms.  They're essentially semi-auto AKs, but with shorter barrels and no stock.  While advertised as compact guns, they're more often than not range toys.  This one is a Draco, the largest, with a 12.5\" barrel."
```

**Review by TheShadowFerret — COMMENTED** [review]
~~Angry Glock noises~~

File: data/json/items/gun/762.json [hunk 5]
```diff
@@ -213,7 +215,97 @@
...
+    "melee_damage": { "bash": 6 }
+  },
+  {
+    "id": "draco_mini",
+    "//": "Based off of https://www.centuryarms.com/mini-draco-pistol.html",
+    "copy-from": "pistol_base",
+    "looks_like": "draco",
+    "type": "GUN",
+    "name": { "str": "Small AK pistol" },
+    "description": "Essentially a semi-auto AK with a short barrel and no stock, this unwieldy \"pistol\" has seen some popularity with civilian shooters and gangbangers alike.  This one has a 7.75\" barrel",
```
Comment (tenmillimaster) [inline data/json/items/gun/762.json:227]: Can you adjust this description to match the others? I prefer your new description to the old one.

File: data/json/itemgroups/Weapons_Mods_Ammo/guns_by_calibre.json [hunk 6]
```diff
@@ -84,6 +84,15 @@
     "id": "guns_pistol_common_10mm",
     "items": [ { "group": "nested_sw_610", "prob": 5 } ]
   },
+  {
+    "type": "item_group",
+    "id": "guns_pistol_common_762_39",
+    "items": [
+      { "group": "nested_draco", "prob": 10 },
```
Comment (tenmillimaster) [inline data/json/itemgroups/Weapons_Mods_Ammo/guns_by_calibre.json:91]: Can you show us a screenshot of the search results?

**Review by tenmillimaster — CHANGES_REQUESTED** [review]
I'm ambivalent on splitting out the draco into different dracos. The nak-9 is a good addition.

File: data/json/items/gun/762.json (hunk 5 above)
Comment (Holli-Git) [reply data/json/items/gun/762.json:227]: Sure

File: data/json/itemgroups/Weapons_Mods_Ammo/guns_by_calibre.json (hunk 6 above)
Comment (Holli-Git) [reply data/json/itemgroups/Weapons_Mods_Ammo/guns_by_calibre.json:91]: image

Comment (tenmillimaster) [reply data/json/itemgroups/Weapons_Mods_Ammo/guns_by_calibre.json:91]: You should be able to look at barrel lengths, too?

**Comment by Holli-Git** [comment]
> I'm ambivalent on splitting out the draco into different dracos. The nak-9 is a good addition.

I did it as the damage difference between the Draco and mini/micro ones is quite steep, and the micro can fit in a large holster with a 30 rounder, while the others cannot fit in a holster regardless of magazine

File: data/json/itemgroups/Weapons_Mods_Ammo/guns_by_calibre.json (hunk 6 above)
Comment (Holli-Git) [reply data/json/itemgroups/Weapons_Mods_Ammo/guns_by_calibre.json:91]: Good to resolve this?

**Review by tenmillimaster — APPROVED** [review]

**Pull request merged**